### ISSUE TITLE
Add Modal Selection Setting Tile

### ISSUE THREAD

**Issue by chitgoks** [yako-dev/flutter-settings-ui#10]
Once user taps on the setting, a modal dialog will show with a list of options to choose from.

**Comment by Akora-IngDKB** [comment]
With the list of options, are you referring to Checkbox or Radio?

**Comment by yadaniyil** [comment]
@chitgoks `onTap()` is now an deprecated parameter. You can use `onPressed(BuildContext context)`.
If you mean [CupertinoActionSheet](https://api.flutter.dev/flutter/cupertino/CupertinoActionSheet-class.html) you can do the following (available in version `settings_ui: ^0.5.0`):

```dart
SettingsTile(
                title: 'Phone number',
                leading: Icon(Icons.phone),
                onPressed: (BuildContext context) {
                  showCupertinoModalPopup(
                    context: context,
                    builder: (context) {
                      return CupertinoActionSheet(
                        title: Text('Make a call'),
                        message: Text('Select phone number'),
                        actions: <CupertinoActionSheetAction>[
                          CupertinoActionSheetAction(
                            child: Text('095-11-40-111'),
                            onPressed: () {
                              Navigator.pop(context);
                            },
                          ),
                          CupertinoActionSheetAction(
                            child: Text('095-11-40-112'),
                            onPressed: () {
                              Navigator.pop(context);
                            },
                          ),
                          CupertinoActionSheetAction(
                            child: Text('095-11-40-123'),
                            onPressed: () {
                              Navigator.pop(context);
                            },
                          ),
                        ],
                        cancelButton: CupertinoActionSheetAction(
                          child: Text('Cancel'),
                          onPressed: () {
                            Navigator.pop(context);
                          },
                        ),
                      );
                    },
                  );
                },
              ),
```
